client/cli/src/lib.rs

```diff
@@ -321,12 +320,13 @@ pub fn init_logger(
 		.with_writer(std::io::stderr)
 		.event_format(logging::EventFormat {
 			timer,
-			ansi: enable_color,
 			display_target: !simple,
 			display_level: !simple,
 			display_thread_name: !simple,
+			enable_color,
 		})
-		.finish().with(logging::NodeNameLayer);
+		.finish()
```

Comment:
Maybe we should add `.with_ansi(enable_color)` here to remove any colors generated from `tracing` itself.

edit: When I added this comment I hadn't read the rest of the code below, I guess this won't be needed.